web/themes/custom/vote_gov/templates/block/block-content--sitewide-alert.html.twig

```diff
@@ -9,7 +9,9 @@
  */
 #}
 
-<div class="usa-alert usa-alert--info">
+{% set variant = content.field_alert_type.0 | render == 'Informational' ? 'info' : 'emergency' %}
+
+<div class="usa-alert usa-alert--{{ variant }}">
```

Comment:
To simplify this and follow the same pattern as other templates. Update the field formatter setting for the field in the admin UI to render the key and not the value. Then update the template like this:

```
{% set classes = [
  'usa-alert',
  content.field_alert_type ? 'usa-alert--' ~ content.field_alert_type | field_value
] %}

<div{{ attributes.addClass(classes)}}>
```